src/sentry/static/sentry/app/views/performance/realUserMonitoring/constants.tsx

```diff
@@ -0,0 +1,44 @@
+import {t} from 'app/locale';
+
+import {Vital, WebVital} from './types';
+
+export const NUM_BUCKETS = 100;
+
+export const PERCENTILE = 0.75;
+
+export const WEB_VITAL_DETAILS: Record<WebVital, Vital> = {
+  [WebVital.FP]: {
+    slug: 'fp',
+    name: t('First Paint'),
+    description: t(
+      'First paint refers to the point at which the first pixel renders on a screen after a user navigates to a web page.'
+    ),
+    failureThreshold: 4000,
+    type: 'duration',
+  },
+  [WebVital.FCP]: {
+    slug: 'fcp',
+    name: t('First Contentful Paint'),
+    description: t(
+      'This is the first moment DOM content such as text or an image gets rendered.'
+    ),
+    failureThreshold: 4000,
+    type: 'duration',
+  },
+  [WebVital.LCP]: {
+    slug: 'lcp',
+    name: t('Largest Contentful Paint'),
+    description: t('This is when the largest content element gets rendered in the page.'),
+    failureThreshold: 4000,
+    type: 'duration',
+  },
+  [WebVital.FID]: {
+    slug: 'fid',
+    name: t('First Input Delay'),
+    description: t(
+      'This is the first moment a user interacts with the page by clicking, scrolling, etc.'
```

Comment:
This description isn't quite right. `FID` is how long the browser takes to respond to the first interaction. It's not like the other measures in terms of being an offset from the start of the pageload.

Reply:
https://web.dev/fid/#what-counts-as-a-first-input

FID doesn't measure scrolling fyi.

Reply:
@k-fish what's a better copy?

Reply:
Yeah I had commented on this earlier, and didn't end up updating it. I'll bring this up again and see if we can get one more pass on these descriptions before merging.

Reply:
This is updated to become:

> First input delay measures the time from a user’s first interaction (such as clicks) to the time when the browser is able to respond to that interaction. Scrolling and zooming are not included in this metric.